package.json

```diff
@@ -24,7 +24,7 @@
     "utils.js"
   ],
   "scripts": {
-    "build": "npm run clean && babel src -d lib --ignore __tests__ --copy-files",
+    "build": "cross-env NODE_ENV=production npm run clean && babel src -d lib --ignore __tests__ --copy-files",
```

Comment:
Will this `cross-env` affect after the `&&`? Do we need `cross-env xxx && cross-env xxx babel xxx`?

Reply:
@tw0517tw You are right.

Reply:
Thanks, I move `cross-env` to the second command which is babel script.